support-figma/extended-layout-plugin/src/ui.html

```diff
@@ -942,6 +942,20 @@
         }
       }, '*');
     };
+    // Double click to exclude node from localization.
+    r.ondblclick = () => {
+      let result = confirm("Are you sure you want to exclude this node from localization?");
+      if (result) {
+        parent.postMessage({
+          pluginMessage: {
+            msg: 'localization-exclude-node',
+            node: nodeId
+          }
+      }, '*');
+      } else {
+        showToast("Action canceled...");
+      }
+    }
```

Comment:
This works, but I find it a little hard to find this feature. There's nothing in the UI to indicate that double clicking would exclude the node. I think some help or instructions are needed if we want to use a double click. Also, once you've excluded a node you can't seem to undo it without clearing out all the localization data. In my opinion, it would be better to have a column (yes I know there are a bit too many columns now) with a checkbox to exclude the node, which would also grey out the rest of the line. This is more a product design issue and I know we'd like to improve other areas as well, but this feature seems harder to use than the others. Maybe get some thoughts from @iamralpht and @timothyfroehlich

Reply:
I changed the double click to use a checkbox in front of the node id.

In order to support the undo action, I ended up including the excluded nodes in the output table like always.

Please take another look and let me know what you think.

Reply:
Ok I like this a lot better. There are some formatting issues when a text node is excluded, but that can be addressed when we get a designer to help...
![Screenshot 2024-07-30 at 9 57 34 PM](https://github.com/user-attachments/assets/65b5f014-3608-405d-b4d3-20d4b9481d69)

Reply:
I made it wrap when it overflows. 
<img width="782" alt="Screenshot 2024-07-31 at 10 49 31" src="https://github.com/user-attachments/assets/e4972849-5da8-4676-a5dd-e5945b786002">